src/components/HelloWorld.vue

```diff
@@ -0,0 +1,207 @@
+<template>
```

Comment:
`HelloWorld`? We should probably rename this file...

Reply:
If, definitely yes, I worked inside the template component, but, if a name change can be made, I have even divided the page into subcomponents (manifest.vue, signBook.vue).